pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/PicklistAnswerDao.java

```diff
@@ -94,13 +80,9 @@ default void assignOptionsToAnswerId(long answerId, List<SelectedPicklistOption>
             selectedIds.add(dto.getId());
         }
 
-        long[] ids = getJdbiPicklistOptionAnswer().bulkInsert(answerId, selectedIds, detailTexts);
+        long[] ids = getAnswerSql().bulkInsertPicklistSelected(answerId, selectedIds, detailTexts);
         if (ids.length != selected.size()) {
             throw new DaoException("Not all selected picklist options were assigned to answer " + answerId);
         }
     }
-
-    default int unassignOptionsFromAnswerId(long answerId) {
```

Comment:
This file is a bit complicated, so I just moved this method to `AnswerSql` and kept the other method here.